### PR TITLE
swiss judgement classification dataset

### PR DESCRIPTION
**Pull request by Akash190104**
Solves issue #542 

Added the swiss judgement classification dataset. Let me know, if anything needs to be added.
## Checklist for adding MMTEB dataset

Reason for dataset addition:

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder] (https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

**Comment by Akash190104** [comment]
ran into a linting issue for a different multihateclassification file.
Would really appreciate the conflict to be resolved.

File: mteb/tasks/Classification/multilingual/SwissJudgementClassification.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        task_subtypes=[
+            "Topic classification",
+            "Political classification",
+            "Claim verification",
+            "Language identification",
+        ],
+        license="CC-BY-4.0",
+        socioeconomic_status="mixed",
+        annotations_creators="expert-annotated",
+        dialect=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/multilingual/SwissJudgementClassification.py:37]:
```suggestion
        dialect=[],
```

File: mteb/tasks/Classification/multilingual/SwissJudgementClassification.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        bibtex_citation="""@misc{niklaus2022empirical,
+    title={An Empirical Study on Cross-X Transfer for Legal Judgment Prediction},
+    author={Joel Niklaus and Matthias Stürmer and Ilias Chalkidis},
+    year={2022},
+    eprint={2209.12325},
+    archivePrefix={arXiv},
+    primaryClass={cs.CL}
+}
+""",
+        n_samples={"train": 59709, "validation": 8208, "test": 17357},
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/multilingual/SwissJudgementClassification.py:48]:
```suggestion
        n_samples={"test": 17357},
```

**Review by KennethEnevoldsen — COMMENTED** [review]
Looks good, a few comments which should be quick to address

Please add points as well

**Comment by Akash190104** [comment]
@KennethEnevoldsen, I have fixed things up based on your reviews. Please let me know if any more changes are necessary.

**Review by KennethEnevoldsen — CHANGES_REQUESTED** [review]

**Comment by KennethEnevoldsen** [comment]
@Akash190104 seems like there still are a few issues with the subsampling

**Comment by Akash190104** [comment]
I made changes based on the review comments.
kept the ```n_samples={"test": 17357}```  but downsampled it later using ```stratified_subsampling```.

Received this error when choosing 2048 sentences:
```ValueError: test_size=2048 should be either positive and smaller than the number of samples 812 or a float in the (0, 1) range```
So went with 2046 samples for 'de' and 'fr' and 810 samples for 'ita'. Hope that isn't a problem. Incase that is, would love to understand how to fix it. Thanks

**Comment by Akash190104** [comment]
Could you please look into this @KennethEnevoldsen ?
I feel this is good to go now.

File: mteb/tasks/Classification/multilingual/SwissJudgementClassification.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        bibtex_citation="""@misc{niklaus2022empirical,
+    title={An Empirical Study on Cross-X Transfer for Legal Judgment Prediction},
+    author={Joel Niklaus and Matthias Stürmer and Ilias Chalkidis},
+    year={2022},
+    eprint={2209.12325},
+    archivePrefix={arXiv},
+    primaryClass={cs.CL}
+}
+""",
+        n_samples={"test": 17357},
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/multilingual/SwissJudgementClassification.py:45]:
```suggestion
        n_samples={"test": 2048},
```

**Review by KennethEnevoldsen — APPROVED** [review]

**Comment by KennethEnevoldsen** [comment]
Thanks for the ping @Akash190104, sorry for the late response. I have fixed a minor issue and enabled automerge

**Pull request merged**